python_modules/dagster/dagster/_daemon/asset_daemon.py

```diff
@@ -440,6 +456,20 @@ def _run_iteration_impl(
                 for location_entry in workspace.get_workspace_snapshot().values()
             }
 
+            eligible_sensors = []
+            for location_entry in workspace_snapshot.values():
+                code_location = location_entry.code_location
+                if code_location:
+                    for repo in code_location.get_repositories().values():
+                        for sensor in repo.get_external_sensors():
+                            if sensor.sensor_type != SensorType.AUTOMATION_POLICY:
+                                continue
```

Comment:
such a nitpick, but i feel like this is slightly easier to parse (for me) if you drop the continue and just append if sensor_type == AUTOMATION_POLICY